profile2/impl/src/java/org/sakaiproject/profile2/logic/ProfilePrivacyLogicImpl.java

```diff
@@ -113,25 +75,14 @@ public boolean savePrivacyRecord(ProfilePrivacy privacy) {
 			return false;
 		}
 		
-		//save
-		if(dao.updatePrivacyRecord(privacy)) {
-			log.info("Saved privacy record for user: " + privacy.getUserUuid()); 
-			
-			//update cache
-			log.debug("Updated privacy record in cache for: " + privacy.getUserUuid());
-			cache.put(privacy.getUserUuid(), privacy);
-			
+		if (dao.updatePrivacyRecord(privacy)) {
```

Comment:
Could be use the return directly?
return dao.updatePrivacyRecord(privacy)

Reply:
updated